src/ShopBundle/objects/db/TShopCategory.class.php

```diff
@@ -648,8 +648,17 @@ public function getTargetPage()
             if (null === $activePortal) {
                 return null;
             }
-            $targetPageId = $activePortal->GetSystemPageId('products');
-            $defaultPage = TdbCmsTplPage::GetNewInstance($targetPageId);
+
+            $systemPageService = $this->getSystemPageService();
+            $defaultSystemPage = $systemPageService->getSystemPage('products', $activePortal);
```

Comment:
What is "default" about it?

Reply:
It is the page we set in code, so it is not configurable. It can be overridden, but if it isn't, this is the default fallback. I would probably have called it something else, but this is old code and I'd rather not question the logic behind it in the context of this PR.